### PR TITLE
fix(docs-infra): return full width highlight to sidenav

### PR DESCRIPTION
**Pull request by sjtrimble**
Sidenav was originally designed to highlight the full width of the selected menu item. This PR returns this functionality.

+ some CSS refactoring

<hr>
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
merge-assistance: flaky windows ci

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 1]
```diff
@@ -81,26 +82,37 @@ mat-sidenav-container div.mat-sidenav-content {
...
   //icons _within_ nav
   .mat-icon {
-    position: absolute;
-    top: 0;
-    right: 0;
-    margin: 4px;
+    // position: absolute;
+    // top: 8px;
+    // right: 8px;
+    // margin: 0;
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_sidenav.scss]: Ooops 😁

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 2]
```diff
@@ -139,33 +156,37 @@ button.vertical-menu-item {
   font-family: $main-font;
   @include font-size(14);
   font-weight: 400;
-  margin-left: 14px;
+  padding-left: 20px;
+  margin: 0;
   transition: background-color 0.2s;
   text-transform: uppercase;
+
+  &.expanded .mat-icon, .level-2.expanded .mat-icon {
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_sidenav.scss]: Two selectors on the same line? 😱

Comment (gkalpak) [reply aio/src/styles/1-layouts/_sidenav.scss]: Fixed it in #31252.

File: aio/src/styles/1-layouts/_sidenav.scss (hunk 1 above)
Comment (gkalpak) [reply aio/src/styles/1-layouts/_sidenav.scss]: Fixed it in #31252.

File: aio/src/styles/1-layouts/_sidenav.scss (hunk 2 above)
Comment (sjtrimble) [reply aio/src/styles/1-layouts/_sidenav.scss]: Thanks :)

**Pull request closed without merging**